### PR TITLE
Better compatibility

### PR DESCRIPTION
**Pull request by hfiguiere**
- keep texmf unpacked (this revert a previous change)
- build asymptote
- build dvisvgm

### CODE REVIEW

File: ghostscript.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# https://flathub.org/builds/#/builders/36/builds/687
+# https://github.com/imagemin/optipng-bin/issues/97#issue-317622868
+# https://discourse.libsdl.org/t/ndk-sdl2-image-libpng-build-error-png-init-filter-functions-neon/24947/3
+build-options:
+  arch:
+    aarch64:
+      cppflags: -DPNG_ARM_NEON_OPT=0
+sources:
+  - type: archive
+    url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs927/ghostscript-9.27.tar.xz
```
Comment (jakobjakobson13) [inline ghostscript.yaml:19]: Could you not just jump to ghostscript 9.53.3? Would this save you some patches?

File: org.freedesktop.Sdk.Extension.texlive.yml [hunk 2]
```diff
@@ -17,10 +18,7 @@ modules:
   - name: texmf
```
Comment (jakobjakobson13) [inline org.freedesktop.Sdk.Extension.texlive.yml:18]: Just for my better understanding: Would it make a difference if this archive got included into the texlive-source module?

File: org.freedesktop.Sdk.Extension.texlive.yml [hunk 3]
```diff
@@ -60,6 +58,37 @@ modules:
...
+  - name: asymptote
+    config-opts:
+      - --bindir=${FLATPAK_DEST}/bin/${FLATPAK_ARCH}-linux
+      - --with-latex=${FLATPAK_DEST}/texmf-dist/tex/latex
+      - --with-context=${FLATPAK_DEST}/texmf-dist/tex/context/third
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/asymptote/asymptote-2.68.src.tgz
+        sha256: e1e85a5db14dc809a43189f85415bd0845bcb9eec7aea5533767838d045b02b2
+  - name: woff2
```
Comment (jakobjakobson13) [inline org.freedesktop.Sdk.Extension.texlive.yml:71]: This seems to be an optional dependency. Do we need it?

File: org.freedesktop.Sdk.Extension.texlive.yml (hunk 2 above)
Comment (hfiguiere) [reply org.freedesktop.Sdk.Extension.texlive.yml:18]: given the time it takes, yes as it allows rebuilding and using the cache. The result is the same.

File: org.freedesktop.Sdk.Extension.texlive.yml (hunk 3 above)
Comment (hfiguiere) [reply org.freedesktop.Sdk.Extension.texlive.yml:71]: it provides woff2 font support.

File: ghostscript.yaml (hunk 1 above)
Comment (hfiguiere) [reply ghostscript.yaml]: done

**Pull request merged**